scripts/segmentation/train.py

```diff
@@ -158,12 +160,20 @@ def __init__(self, args, logger):
 
         # create network
         if args.model_zoo is not None:
-            model = get_model(args.model_zoo, pretrained=True)
+            model = get_model(args.model_zoo, norm_layer=args.norm_layer,
```

Comment:
Do we need to keep the `pretrained=True` since these are model_zoo models? Otherwise, the `get_model ` here will be just like `get_segmentation_model ` below.

Reply:
Will add a `--pretrained` option.